python/test/pyError_TEST.py

```diff
@@ -51,6 +52,18 @@ def test_value_construction_without_file(self):
     self.assertFalse(error.line_number())
 
 
+  def test_deepcopy(self):
+    error = Error(Error.ErrorCode.FILE_READ, "Unable to read a file")
+
+    error2 = copy.deepcopy(error)
+    self.assertEqual(error.is_valid(), error2.is_valid())
+    self.assertEqual(error.code(), error2.code())
+    self.assertEqual(error.message(), error2.message())
+    self.assertEqual(error.xml_path(), error2.xml_path())
+    self.assertEqual(error.file_path(), error2.file_path())
+    self.assertEqual(error.line_number(), error2.line_number())
```

Comment:
nit: for completeness you could modify `error2` and then confirm that it doesn't affect `error`, but it's not necessary